meetings/submissions/migrations/0001_initial.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.9.7 on 2016-07-25 14:16
+
+from __future__ import unicode_literals
 
 from __future__ import unicode_literals
```

Comment:
Yeah this was a mistake when I was fixing the merge conflicts, I'll take out one of the imports.